data/volunteers.js

```diff
@@ -9,11 +9,36 @@ export default [
     title: 'Front-End Developer',
     image: 'simi-damani.jpg',
   },
+  {
+    name: 'Nick Holden',
+    title: 'Software Engineer',
```

Comment:
I made this title up for myself. I wonder if it makes sense to consolidate some of these titles (for example, is differentiating between "software engineer," "senior software engineer," and "front-end developer" helpful?). Not a high priority for me though if it doesn't bother anyone else.